src/corehost/cli/runtime_config.cpp

```diff
@@ -112,3 +112,18 @@ bool runtime_config_t::get_portable() const
 {
     return m_portable;
 }
+
+void runtime_config_t::config_kv(std::vector<std::string>* keys, std::vector<std::string>* values) const
+{
+    for (const auto& kv : m_properties)
+    {
+        keys->push_back(pal::to_stdstring(kv.first));
+        values->push_back(pal::to_stdstring(kv.second));
+    }
+
+    if (m_properties.count(_STRINGIFY(System_GC_Server)) == 0)
+    {
+        keys->push_back(System_GC_Server);
+        values->push_back("true"); // Default true
```

Comment:
Server GC can't be on by default, it's not applicable for regular apps. The request was to turn it on by default for ASP.NET apps in the project template.